src/main/kotlin/edu/cornell/cs/apl/viaduct/syntax/NameMap.kt

```diff
@@ -6,7 +6,7 @@ import kotlinx.collections.immutable.PersistentMap
 import kotlinx.collections.immutable.persistentMapOf
 
 /** A persistent map from [Name]s to [Data]. */
-internal class NameMap<N : Name, Data>
+class NameMap<N : Name, Data>
```

Comment:
Why drop internal? All name analysis is done for us in `NameAnalysis` so no one outside the module needs to know this exists...